### PR TITLE
mitmdump: Add Basic Support for TCP Flows

### PR DESCRIPTION
**Pull request by mhils**
This PR introduces support for saving TCP flows in dumpfiles. It's mitmdump only for now.

### CODE REVIEW

File: mitmproxy/models/tcp.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    def __repr__(self):
+        return "{direction} {content}".format(
+            direction="->" if self.from_client else "<-",
+            content=repr(self.content)
+        )
+
+
+class TCPFlow(Flow):
+    """
+    A SSHFlow is a simplified representation of an SSH session.
```
Comment (Kriechi) [inline mitmproxy/models/tcp.py]: `s/SSH/TCP/g`

**Comment by Kriechi** [comment]
This looks really great so far!

Still working on it? 
Would be great if the tests pass reliably.

**Comment by mhils** [comment]
This is ready for a final review I think! :smiley:

**Comment by Kriechi** [comment]
LGTM! :shipit:

**Pull request merged**